src/preprocessing/passes/bv_to_bool.cpp

```diff
@@ -171,9 +169,15 @@ Node BVToBool::convertBvTerm(TNode node)
   ++(d_statistics.d_numTermsLifted);
 
   Kind kind = node.getKind();
-  if (kind == kind::ITE)
+  if (kind == kind::ITE || kind == kind::BITVECTOR_ITE)
   {
-    Node cond = liftNode(node[0]);
+    Node cond = node[0];
+    // Translate bit-vector condition to an equality to (_ bv1 1)
+    if (kind == kind::BITVECTOR_ITE)
+    {
+      cond = NodeManager::currentNM()->mkNode(kind::EQUAL, cond, d_one);
+    }
```

Comment:
What happens if we have e.g.,

```
(BITVECTOR_ITE (BITVECTOR_COMP ...) x y)
```

`BITVECTOR_COMP` gets converted to `EQUAL`. Can it happen that you will compare an `EQUAL` against  `d_one`?